subprojects/base-services/src/main/java/org/gradle/util/internal/EncryptionAlgorithm.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This was originally placed here so it could be seen by encrypting/decrypting encoder/decoder implementations I had added to the `messaging` subproject.